README.md

```diff
@@ -153,6 +153,7 @@ NOTE: You do not need to specify *end* segment - it will be added automatically.
 |--------|-------|-------|
 |`BULLETTRAIN_VIRTUALENV_BG`|`yellow`|Background color
 |`BULLETTRAIN_VIRTUALENV_FG`|`white`|Foreground color
+|`BULLETTRAIN_VIRTUALENV_NAME`|` %m`|Text to show after the prefix (use %m for virtualenv name)
```

Comment:
This is needed in order to avoid overlapping of text and the emoji prefix, thus reproducing the current default behavior.

Another way to go could be to delete totally the prefix support, and to include the emoji itself in the venv name property. Thus, default value could be `'🐍 %m'`.

This is IMHO the proper way to go, i will update.